{
    "CVE_data_meta": {
        "ASSIGNER": "secalert@redhat.com",
        "ID": "CVE-2018-10930",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "Red Hat"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "A flaw was found in RPC request using gfs3_rename_req in glusterfs server. An authenticated attacker could use this flaw to write to a destination outside the gluster volume."
            }
        ],
        "metrics": [
            {
                "cvss-v_3_0": {
                    "vectorString": "6.5/CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:N",
                    "version": "3.0"
                }
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "CWE-ID": "CWE-20",
                        "description": "CWE-20",
                        "lang": "eng",
                        "type": "CWE"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10930",
                "refsource": "CONFIRM",
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10930"
            },
            {
                "name": "RHSA-2018:2607",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2018:2607"
            },
            {
                "name": "https://review.gluster.org/#/c/glusterfs/+/21068/",
                "refsource": "CONFIRM",
                "url": "https://review.gluster.org/#/c/glusterfs/+/21068/"
            },
            {
                "name": "[debian-lts-announce] 20180920 [SECURITY] [DLA 1510-1] glusterfs security update",
                "refsource": "MLIST",
                "url": "https://lists.debian.org/debian-lts-announce/2018/09/msg00021.html"
            },
            {
                "name": "RHSA-2018:2608",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2018:2608"
            },
            {
                "name": "RHSA-2018:3470",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2018:3470"
            },
            {
                "name": "GLSA-201904-06",
                "refsource": "GENTOO",
                "url": "https://security.gentoo.org/glsa/201904-06"
            },
            {
                "name": "openSUSE-SU-2020:0079",
                "refsource": "SUSE",
                "url": "http://lists.opensuse.org/opensuse-security-announce/2020-01/msg00035.html"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}